analyses/cell-type-nonETP-ALL-03/scripts/writeout_submission.R

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env Rscript
+
+library(Seurat)
+library(ggplot2)
+
+writeout <- function(ind.lib, ct.colors = ct_color, project.ID = projectID, n.row = 1){
+  seu <- readRDS(file.path(out_loc,"results/rds",paste0(ind.lib,".rds")))
+  voi <- c('newB.copykat.pred','sctype_classification')
+  changeName.voi <- c('tumor_cell_classification','cell_type_assignment')
+  tryCatch({
+    voi_df <- data.frame(FetchData(seu, vars = voi)) |> tibble::rownames_to_column(var = "cell_barcode")
+  }, error=function(e){})
+  colnames(voi_df)[2:length(voi_df)] <- changeName.voi[match(colnames(voi_df)[2:length(voi_df)],voi)]
+  final.df <- data.frame(scpca_sample_id=rep(project.ID, nrow(voi_df)), voi_df, 
+                         CL_ontology_id=gene.df$ontologyID[match(voi_df$cell_type_assignment,gene.df$cellName)])
+  write.table(final.df, sep = "\t", quote = F, row.names = F,
+              file = file.path(out_loc,"results/submission_table",paste0(ind.lib,"_metadata.tsv")))
```

Comment:
Please include creating `results/submission_table` earlier in the script. I believe this is the cause of the current CI failure.